### PR TITLE
OpenJS Fdn blog post

### PR DESCRIPTION
**Pull request by knolleary**
Closes: https://github.com/flowforge/content/issues/25

### CODE REVIEW

File: src/blog/2022/12/flowforge-joins-openjs-foundation.md [hunk 1]
```diff
@@ -0,0 +1,33 @@
+---
+title: FlowForge joins the OpenJS Foundation
```
Comment (ZJvandeWeg) [inline src/blog/2022/12/flowforge-joins-openjs-foundation.md:2]: Do we join them? Or support them?

**Review by ZJvandeWeg — APPROVED** [review]

File: src/blog/2022/12/flowforge-joins-openjs-foundation.md (hunk 1 above)
Comment (knolleary) [reply src/blog/2022/12/flowforge-joins-openjs-foundation.md:2]: We have joined as members.

Comment (knolleary) [reply src/blog/2022/12/flowforge-joins-openjs-foundation.md:2]: Could change to FlowForge Inc. joins...?

Comment (ZJvandeWeg) [reply src/blog/2022/12/flowforge-joins-openjs-foundation.md:2]: I think that's better indeed.

```suggestion
title: FlowForge Inc. joins the OpenJS Foundation
```

Comment (knolleary) [reply src/blog/2022/12/flowforge-joins-openjs-foundation.md:2]: > Ok, it sounds like we've donated FlowForge. We donated, I think it's different.

Changing to: 'FlowForge Inc becomes a member of the OpenJS Foundation' to remove any confusion

**Pull request merged**